base/src/actions.rs

```diff
@@ -54,18 +79,25 @@ impl Model {
             .cell_formula(sheet, source_row, source_column)?
             .unwrap_or_else(|| source_cell.get_text(&self.workbook.shared_strings, &self.language));
         self.set_user_input(sheet, target_row, target_column, formula_or_value);
-        self.workbook
-            .worksheet_mut(sheet)?
-            .set_cell_style(target_row, target_column, style);
+        self.workbook.worksheet_mut(sheet)?.set_cell_style(target_row, target_column, style);
         self.delete_cell(sheet, source_row, source_column)?;
         Ok(())
     }
 
+    /// Inserts one or more new columns into the model at the specified index.
+    ///
+    /// This method shifts existing columns to the right to make space for the new columns.
+    ///
+    /// # Arguments
+    ///
+    /// * `sheet` - The sheet number to retrieve columns from.
+    /// * `column` - The index at which the new columns should be inserted.
+    /// * `column_count` - The number of columns to insert.
```

Comment:
This is great! 
Note that this is a public function and the sort of functions we should be documenting first.

I started doing this at the lexer:

https://github.com/ironcalc/IronCalc/blob/67e8b0ce145930b02048e2e5da4ed92384c9f859/base/src/expressions/lexer/util.rs#L17-L49

NB: no action required, we will add examples in a different PR

Reply:
Sounds good.  I'll focus on the public functions initially then. 🙂